modules/ui/index.js

```diff
@@ -0,0 +1,111 @@
+import { ui } from './ui';
+import { presetObj } from './preset/index';
+import {
+    Account,
+    Attribution,
+    Background,
+    cmd,
+    Commit,
+    confirm,
+    Conflicts,
+    Contributors,
+    Disclosure,
+    EntityEditor,
+    FeatureInfo,
+    FeatureList,
+    flash,
+    FullScreen,
+    Geolocate,
+    Help,
+    Info,
+    Inspector,
+    intro,
+    Lasso,
+    Loading,
+    MapData,
+    MapInMap,
+    modal,
+    Modes,
+    Notice,
+    preset,
+    PresetIcon,
+    PresetList,
+    RadialMenu,
+    RawMemberEditor,
+    RawMembershipEditor,
+    RawTagEditor,
+    Restore,
+    Save,
+    Scale,
+    SelectionList,
+    Sidebar,
+    SourceSwitch,
+    Spinner,
+    Splash,
+    Status,
+    Success,
+    TagReference,
+    Toggle,
+    tooltipHtml,
+    UndoRedo,
+    ViewOnOSM,
+    Zoom
+} from './core/index';
+export {ui};
+
+ui.preset = preset;
+Object.keys(presetObj).map(function(p) {
+    ui.preset[p] = presetObj[p];
+});
+
+// Need to do for test cases
+ui.intro = intro;
```

Comment:
I had to add these keys to ui as `tests` still use it.
If we can create  separate build for tests and have a node.env variable to conditionally bundle this code. We can very much remove it for our production build.